packages/app-project/stores/User/UserPersonalization/YourStats/YourStats.spec.js

```diff
@@ -67,7 +67,9 @@ describe('Stores > YourStats', function () {
     let clock
 
     before(function () {
-      clock = sinon.useFakeTimers({ now: new Date(2019, 9, 1, 12), toFake: ['Date'] })
+      // Set the local clock to 1am on Tuesday 1 October 2019, UTC.
+      // Stats should be shown for Monday 30 September 2019, local time.
+      clock = sinon.useFakeTimers(new Date('2019-09-30T20:00:00-05:00'))
```

Comment:
This won't set the clock timezone to `-05:00`. `new Date()` always runs in the time zone of the machine that's running the code.

Reply:
Methods like `date.getDay()` always return values in the OS time zone too.